docs/content/concepts/resources.mdx

```diff
@@ -63,7 +76,26 @@ def op_requires_resources(context):
     context.resources.database.execute_query(CREATE_TABLE_1_QUERY)
 ```
 
-## Providing Resources to a Job
+### In assets
+
+As assets are functionally similar to ops, assets also use resource keys to access resources:
+
+<!-- This hard-coded example is temporary. -->
+
+```python
```

Comment:
This example is temporary. Another example on #8119 ([link](examples/docs_snippets/docs_snippets/concepts/assets/asset_config.py)) could also work here, unless we want to do something else. I was planning to pull that in here after the PR is merged.